src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -431,6 +431,16 @@ export class SbbFormField implements ComponentInterface {
     await this._checkAndUpdateInputEmpty();
   }
 
+  /** Manually clears the input value. */
+  @Method() public async clear(): Promise<void> {
+    if (this._input.tagName !== 'INPUT') {
+      return;
+    }
```

Comment:
What would it need to also support select?

Reply:
Lukas indicated to me to start by supporting only the simple inputs. For the select, I think we need to define a clear approach for a possible empty state first.

Reply:
thats basically ok, could you create a Draft (card) to our board, so we will not forget to maybe design something like this? At least it is still possible for a customer to add it's own click handler to the button